src/main/java/org/jenkinsci/main/modules/sshd/SSHD.java

```diff
@@ -106,6 +106,12 @@ public synchronized void start() throws IOException, InterruptedException {
 
         sshd.setPublickeyAuthenticator(new PublicKeyAuthenticatorImpl());
 
+        // Allow to configure idle timeout with a system property
+        String idleTimeout = System.getProperty(SSHD.class.getName() + "." + IDLE_TIMEOUT_KEY);
```

Comment:
NIT: IMHO would be great to have a pre-verification ant fallback instead of just passing string.